samples/shields/x_nucleo_iks02a1/microphone/sample.yaml

```diff
@@ -1,7 +1,7 @@
 sample:
   name: X-NUCLEO-IKS02A1 sensor shield
 tests:
-  test:
+  sample.shields.x_nucleo_iks02a1:
```

Comment:
Looking at what samples/shield/x_nucleo_iks01a3 is doing I thin we can put here:
sample.shields.x_nucleo_iks02a1.microphone

And change the equivalent in standard and sensorhub as:
  sample.shields.x_nucleo_iks02a1.standard
and
  sample.shields.x_nucleo_iks02a1.sensorhub